src/tools/storage-perf/StoragePerfTool.cpp

```diff
@@ -34,13 +35,21 @@ class Perf {
         LOG(INFO) << "Total threads " << FLAGS_threads
                   << ", qpsPerThread " << qpsPerThread
                   << ", task interval ms " << interval;
+        auto metaAddrsRet = nebula::network::NetworkUtils::toHosts(FLAGS_meta_server_addrs);
+        if (!metaAddrsRet.ok() || metaAddrsRet.value().empty()) {
+            LOG(ERROR) << "Can't get metaServer address, status:" << metaAddrsRet.status()
+                       << ", FLAGS_meta_server_addrs:" << FLAGS_meta_server_addrs;
+            return -1;
+        }
         std::vector<std::unique_ptr<thread::GenericWorker>> threads;
         for (int32_t i = 0; i < FLAGS_threads; i++) {
             auto t = std::make_unique<thread::GenericWorker>();
             threads.emplace_back(std::move(t));
         }
         threadPool_ = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_io_threads);
-        client_ = std::make_unique<StorageClient>(threadPool_, nullptr);
+        mClient_ = std::make_unique<meta::MetaClient>(threadPool_, metaAddrsRet.value());
+        CHECK(mClient_->waitForMetadReady());
+        client_ = std::make_unique<StorageClient>(threadPool_, mClient_.get());
```

Comment:
line 69. Due to multiple threading, is it possible the main thread skip 1000*X after some sleep? e.g. 998->sleep->1002